addrmgr/addrmanager.go

```diff
@@ -70,7 +70,7 @@ type serializedAddrManager struct {
 }
 
 type localAddress struct {
-	na    *wire.NetAddress
+	na    *wire.NetAddressV2
```

Comment:
Is this change backwards compatible w/ the _existing_ `peers.json` we keep on disk, or are we assuming we'll just do a one time translation (next time the new version reads the file)?